JavaScript/syntax_test_js.js

```diff
@@ -198,8 +234,12 @@ not_a_comment;
     '// /* not a comment'() {},
 //  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ -comment() {}
 
+    '// /* not a comment'() {},
+//  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ -comment() {}
+
```

Comment:
How is this different than the assertions right above?

Reply:
It almost certainly isn't. Looks like it was added four months ago, probably by mistake.